html/semantics/embedded-content/the-model-element/attributes/global/draggable.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      rel="help"
+      href="https://html.spec.whatwg.org/multipage/embedded-content-other.html#the-model-element"
+    />
+    <title>Test for the functionality of the draggable attribute</title>
+
+    <style>
+      model,
+      img {
+        background-color: rgb(255, 0, 0);
+      }
+    </style>
+  </head>
+
+  <body>
+    <model draggable="true" width="200" height="200">
+      <source src="../../resources/chromeball.usdz" type="model/vnd.usdz+zip" />
+      <source src="../../resources/chromeball.glb" type="model/gltf-binary" />
+    </model>
+  </body>
+  <!-- Include the test harness -->
+
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+
+  <script>
+    const model = document.querySelector("model");
+
+    model.addEventListener("dragstart", (dragEvent) => {
+      model.style.backgroundColor = "rgb(0, 255, 0)";
+    });
+
+    model.addEventListener("dragend", (dragEvent) => {
+      model.style.backgroundColor = "rgb(0, 0, 255)";
+    });
+
+    promise_test(async (t) => {
+      const modelRect = model.getBoundingClientRect();
+      const initialStyle = getComputedStyle(model);
+      assert_equals(
+        initialStyle.getPropertyValue("background-color"),
+        "rgb(255, 0, 0)",
+        "model has valid initial bgcolor"
+      );
+
+      await new test_driver.Actions()
+        .pointerMove(modelRect.x + 10, modelRect.y + 10)
+        .pointerDown()
+        .addTick()
+        .pointerMove(modelRect.x + 100, modelRect.y + 10)
+        .send();
```

Comment:
Then you probably want: 

```suggestion
      await new test_driver.Actions()
        .pointerMove(modelRect.x + 10, modelRect.y + 10)
        .pointerDown()
        .addTick()
        .send();
      await dragStartPromise;
```

And the same with the up... await the event to fire.